docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,628 @@
+
+.. list-table:: Screening branches
+   :header-rows: 1
+
+   * - Branch
+     - Sex
+     - Age
+     - Screening technology
+     - Co-test result
+     - Co-test sensitivity
+     - Co-test specificity
+     - Follow-up test
+     - Follow-up test sensitivity
+     - Follow-up test specificity
+   * - A
+     - Female
+     - 21-65
+     - Cytology plus HPV test, every 3 years
+     - HPV negative, Cytology negative
+     - 17.4%
+     - 100%
+     - Cytology, every 3 years
+     - 65.9% (95%CI 54.9 to 75.3)
+     - 100%
+   * - B
+     - Female
+     - 21-65
+     - Cytology plus HPV test, every 3 years
+     - HPV positive, Cytology negative
+     - 23.5%
+     - 100%
+     - Cytology, every year
+     - 65.9% (95%CI 54.9 to 75.3)
+     - 100%
+   * - C
+     - Female
+     - 21-65
+     - Cytology plus HPV test, every 3 years
+     - HPV negative, Cytology positive
+     - 5.9%
+     - 100%
+     - 
+     - 
+     - 
+   * - D
+     - Female
+     - 21-65
+     - Cytology plus HPV test, every 3 years
+     - HPV positive, Cytology positive
+     - 53.2%
+     - 100%
+     - 
+     - 
+     - 
+   * - E
+     - Female
+     - <21 or 65+
+     - No screening
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+   * - F
+     - Male
+     - Any
+     - No screening
+     - 
+     - 
+     - 
+     - 
+     - 
+     - 
+
+.. note::
+ 
+ - Co-test (cytology plus HPV test) is not highly recommended for women under 
+   30 according to guidelines from American Cancer Society and U.S. Preventive Services Task Force. To simply our screening algorithm, we decided to set initial age of the co-test to be the same as initial age of cytology alone, 
+   namely 21 years old in our simulation.
+ - The stand-in value of co-test sensitivity was extracted from Schiffman et al. 
+   2018.
+ - For now, follow-up test is not applicable with positive Cytology results. 
+   Though colposcopy is recommended after diagnosis of benign/invasive cervical cancer, we could use colposcopy referral rate to determine treatment coverage for benign cervical cancer. 
+ - Specificity for both co-test and cytology alone are set to 100% in our model. 
+
+In initialization, We assume that
+ - No one has prior knowledge of their disease status (and HPV status) on day one 
+   of the simulation.
+ - All simulants are buying insurance on day one of the simulation.
+ - For simulants in cervical cancer (CC) state regardless of detection, they have 
+   a transition rate of 0.1 (per person-year) of moving into a recovered (R) state; this results in an average duration in state CC of 10 years. People in state CC and R follow exactly the same screening algorithm, namely branch A, B, C, or D depending on their age, sex, and co-test result. Simulants do not ever make a second cervical cancer claim, therefore the negative screening results were expected for those in R state in order to avoid double counting the CI claim from detected cervical cancer.
```

Comment:
I think it would be good to add some justification for the duration of 10 years - either literature or GBD assumptions

Reply:
It's clearly documented in GBD 2017 neoplasm/cause model doc, i will throw a link there for easy reference.